### ISSUE TITLE
Test line is laid at the very edge of the build plate - BL A1

### ISSUE THREAD

**Issue by otede** [siganberg/3D-printing#2]
Take a look at the photo. It's not catastrophic but still. And thank you for the SANE startup! :)
![IMGoc_20241104_110712](https://github.com/user-attachments/assets/3be042dd-ed84-41f1-bb96-78050615680f)

**Comment by siganberg** [comment]
Yeah, I noticed that but I kept it because it actually makes it much easier to peel or remove plus you have the maximum printable space.

**Comment by siganberg** [comment]
If you wanna adjust it just change the Y value on this line. You can make it 0 or 1 instead of negative.  

![image](https://github.com/user-attachments/assets/ef792e82-8b66-4476-a4ec-616ecff21b67)

Or you can flip your PEI sheet. 😄

**Comment by otede** [comment]
Thanks!
